pkgs/by-name/ha/harlequin/package.nix

```diff
@@ -47,9 +62,9 @@ python3Packages.buildPythonApplication rec {
       questionary
       rich-click
       sqlfmt
-      textual
-      textual-fastdatatable
-      textual-textarea
+      textual_5_3_0
+      (textual-fastdatatable.override { textual = textual_5_3_0; })
```

Comment:
Please use python package yet overrides like documented in the manual to properly override the dependency everywhere.